CMakeLists.txt

```diff
@@ -406,14 +406,18 @@ target_include_directories(avif
                            PUBLIC $<BUILD_INTERFACE:${libavif_SOURCE_DIR}/include>
                                   $<INSTALL_INTERFACE:include>
                            PRIVATE ${AVIF_PLATFORM_INCLUDES} ${AVIF_CODEC_INCLUDES})
+set(AVIF_PC_EXTRA_DEFINITION "")
 if(BUILD_SHARED_LIBS)
     target_compile_definitions(avif PRIVATE -DAVIF_BUILDING_SHARED_LIBS)
+    if(WIN32)
+        target_compile_definitions(avif INTERFACE -DAVIF_DLL)
```

Comment:
We can omit "-D" in target_compile_definitions:

        target_compile_definitions(avif INTERFACE AVIF_DLL)

Note: The "Step-by-step guide" suggests making AVIF_DLL a PUBLIC definition rather than an INTERFACE definition. Either way will work. We can start with INTERFACE and only change it to PUBLIC when libavif itself needs to know whether it is a shared library or not.